src/praisonai-code/praisonai_code/cli/configuration/paths.py

```
@@ -161,6 +161,23 @@ def get_project_config_path(project_root: Optional[Path] = None) -> Path:
+
+    Delegates to the core ``praisonaiagents.paths.get_state_dir`` so the wrapper
+    and SDK never diverge: honours ``PRAISONAI_HOME`` / an existing single root,
+    and otherwise resolves to ``$XDG_STATE_HOME/praisonai`` (else
+    ``~/.local/state/praisonai``). Falls back to the canonical home root if the
+    core package is unavailable for any reason.
+    """
+    try:
+        from praisonaiagents.paths import get_state_dir as _core_state_dir
+
+        return _core_state_dir()
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)